root/application/src/main/java/at/fhv/itb5c/rmi/application/controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package at.fhv.itb5c.rmi.application.controller;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+import at.fhv.itb5c.commons.dto.IUser;
+import at.fhv.itb5c.commons.dto.IUserController;
+import at.fhv.itb5c.model.entity.User;
+import at.fhv.itb5c.rmi.application.dto.UserConverter;
+
+public class UserController implements IUserController {
+
+	@Override
+	public IUser createUser(IUser user) {
+		User userEntity = UserConverter.toEntity(user);
+
+		// TODO not nice!
+		EntityManagerFactory emf = Persistence.createEntityManagerFactory("persistence.odb");
+		EntityManager em = emf.createEntityManager();
+		em.getTransaction().begin();
+		em.persist(userEntity);
+		em.getTransaction().commit();
```

Comment:
do we know if the commit completed successfully? or is an exception thrown if the transaction can not be committed? We don't want to return the user if it could not be stored in the database.